v1.0/openapi3/schemas/Name.yaml

```diff
@@ -1,4 +1,13 @@
 title: Name
 type: string
 pattern: '^(?!\s*$)[\p{L}\p{Nd} .,''-]{1,128}$'
-description: "The API data type Name is a JSON String, restricted by a regular expression to avoid characters which are generally not used in a name.\nRegular Expression - The regular expression for restricting the Name type is \"^(?!\\s*$)[\\w .,'-]{1,128}$\". The restriction does not allow a string consisting of whitespace only, all Unicode characters are allowed, as well as the period (.) (apostrophe (‘), dash (-), comma (,) and space characters ( ).\n\n\n**Note:** In some programming languages, Unicode support must be specifically enabled. For example, if Java is used, the flag UNICODE_CHARACTER_CLASS must be enabled to allow Unicode characters."
+description: |
+  The API data type Name is a JSON String, restricted by a regular expression
+  to avoid characters which are generally not used in a name.\n
+  Regular Expression - The regular expression for restricting the
+  Name type is \"^(?!\\s*$)[\\w .,'-]{1,128}$\". The restriction does not
```

Comment:
Again, do we still need the escape character here?

Reply:
Unsure. Let's leave it as is for now.

Reply:
Okok 👍